src/pages/settings/Wallet/TransferBalancePage.tsx

```diff
@@ -156,12 +156,10 @@ function TransferBalancePage({bankAccountList, fundList, userWallet, walletTrans
                 titleKey="notFound.pageNotFound"
                 subtitleKey="transferAmountPage.notHereSubTitle"
                 linkKey="transferAmountPage.goToWallet"
-                onLinkPress={() => Navigation.goBack(ROUTES.SETTINGS_WALLET)}
```

Comment:
To cover the text button of the not found page case. The default props is Navigation.dismissModal

https://github.com/Expensify/App/assets/50919443/e1a9d979-1740-4081-b890-4e68092ee710